api/src/opentrons/protocol_engine/commands/command_unions.py

```diff
@@ -5,14 +5,41 @@
     AddLabwareDefinition,
     AddLabwareDefinitionRequest,
     AddLabwareDefinitionResult,
+    AddLabwareDefinitionCommandType,
+)
+
+from .aspirate import Aspirate, AspirateRequest, AspirateResult, AspirateCommandType
+
+from .dispense import Dispense, DispenseRequest, DispenseResult, DispenseCommandType
+
+from .drop_tip import DropTip, DropTipRequest, DropTipResult, DropTipCommandType
+from .load_labware import (
```

Comment:
```suggestion
from .drop_tip import DropTip, DropTipRequest, DropTipResult, DropTipCommandType

from .load_labware import (
```

Whitespace nitpick for consistency with the rest of this file.